CHANGELOG

```diff
@@ -4,6 +4,7 @@
       data from the loaded file.
     * Added option "synchronize_simulation" to allow for synchronizing 
       simulation with (scaled) real-time.
+    * Bugfix in fmi20_bouncing_ball_native.py, would trigger a step_event every timestep.
```

Comment:
I might be a bit pedantic here but I would rephrase _bugfix_ to something else, such as `Resolved a bug in the example fmi20_bouncing_ball_native where a time step is triggered for every step event.`

Reply:
I like that suggestion, changed it to:

`Resolved a bug in the example fmi20_bouncing_ball_native where a step-event is triggered in every timestep.`